NOTES-Android.md

```diff
@@ -1,6 +1,5 @@
-
- NOTES FOR ANDROID PLATFORMS
- ===========================
+NOTES FOR ANDROID PLATFORMS
+===========================
```

Comment:
This is one spot where I think mdl is over zealous

Reply:
> This is one spot where I think mdl is over zealous

I though so too, but excluding the respective rule was for some reason not sufficient.
Well, and un-indenting the first two lines in a couple of `.md` files was no big deal.